pages/instances/how-to/use-flexips.mdx

```diff
@@ -82,4 +82,161 @@ When you no longer want one of your existing flexible IP addresses, you can dele
+
+Prior to routed IPs, IPv6 addresses were statically routed to the Hypervisors, which means your IP changed depending on which Hypervisor your Instance was running on.
+
+### IPv6 prefix and SLAAC
+
+Now let's dive into the routed IPv6. Whenever you book a `routed_ipv6` IP type, you get assigned a full IPv6 prefix of 64bits length. According to the [section 2.4](https://datatracker.ietf.org/doc/html/rfc4291#section-2.4) of the RFC4291, except a few prefixes, anything is a Global Unicast  address type. Today's [best practice](https://www.ripe.net/publications/docs/ripe-690/#4-1-1---64-prefix-from-a-dedicated-pool-of-ipv6-prefixes) is to provide a /64 prefix.
+
+Another reason we decided to provide you with a /64 prefix is to be able to use [SLAAC](https://datatracker.ietf.org/doc/html/rfc4862) as the configuration mechanism. But SLAAC has been designed to work [exclusively with /64 prefixes](https://notes.networklessons.com/ipv6-prefix-length-used-for-slaac).
+
+We chose SLAAC over DHCPv6 because it doesn't require any client-side configuration in most of the cases, allowing the feature to work with old instances too.
+
+### Limitations
+
+As you may have already noticed, there are some limitations to this design that we tried to minimize.
+
+The first one is that, in order not to break already deployed SDKs, or CLIs, we had to introduce a new `prefix` field to the `/ips` endpoint:
+
+```
+        {
+            "address": null,
+            "id": "153e451e-866b-4f58-9241-3bdbe98a0175",
+            "ipam_id": "a81665be-0059-431f-bf44-8d7bdf1dac01",
+            "organization": "8e214357-66e2-43ee-8898-0da03fd869e2",
+            "prefix": "2025:bd8:911::/64",
+            "project": "8e214357-66e2-43ee-8898-0da03fd869e2",
+            "reverse": null,
+            "server": null,
+            "state": "detached",
+            "tags": [],
+            "type": "routed_ipv6",
+            "zone": "fr-par-1"
+        }
+```
+
+On the other hand, the `/servers/<id>` endpoint provides the `public_ip` (deprecated) field and its replacement `public_ips`. This struct also contain an address field of type IP that doesn't accept a prefix:
+
+```
+[
+  {
+    "id": "e37b038d-c5f7-4fc7-8ac2-407807f723dd",
+    "address": "2025:bd8:911:46:dc00:ff:fe08:ab0c",
+    "dynamic": false,
+    "gateway": "fe80::dc00:ff:fe08:ab0c",
+    "netmask": "64",
+    "family": "inet6",
+    "provisioning_mode": "slaac",
+    "tags": [],
+    "state": "attached",
+    "ipam_id": "45791fdb-c576-4447-985b-19b45740714b"
+  }
+]
+```
+
+Instead, we return the SLAAC address based on the EUI-64 of the interface.
+
+One consequence is that, if you move your `routed_ipv6` **prefix** to another instance (hence, with another MAC address), the `public_ips` list will show you a **different** address because its EUI-64 will differ.
```

Comment:
```suggestion
One consequence is that, if you move your `routed_ipv6` **prefix** to another Instance (hence, with another MAC address), the `public_ips` list will show you a **different** address because its EUI-64 will differ.
```